### ISSUE TITLE
Revanced Manager Crash

### ISSUE THREAD

**Issue by Bluelytning16** [ReVanced/revanced-manager#1636]
### Bug description

https://github.com/ReVanced/revanced-manager/assets/114093892/82a24fc9-ced5-45ab-91c9-3ee8cad54de1

### Version of ReVanced Manager and version & name of application you tried to patch

Revanced 1.18
Youtube latest compatible 18.45.43

### Installation type

Non-root

### Device logs

```shell
--------- beginning of system
01-13 11:45:47.199 19550 19550 D ActivityThread: handleBindApplication()++ app=app.revanced.manager.flutter
01-13 11:45:47.240 19550 19550 D LoadedApk: LoadedApk::makeApplication() appContext.mOpPackageName=app.revanced.manager.flutter appContext.mBasePackageName=app.revanced.manager.flutter
01-13 11:45:47.258 19550 19550 D ActivityThread: handleBindApplication() --
--------- beginning of main
01-13 16:46:38.674 31709 31709 E manager.flutte: Not starting debugger since process cannot load the jdwp agent.
01-13 16:46:38.691 31709 31709 E USNET   : USNET: appName: app.revanced.manager.flutter
01-13 16:46:38.692 31709 31709 D ProcessState: Binder ioctl to enable oneway spam detection failed: Invalid argument
01-13 16:46:38.703 31709 31709 D ActivityThread: setConscryptValidator
01-13 16:46:38.704 31709 31709 D ActivityThread: setConscryptValidator - put
01-13 16:46:38.733 31709 31709 D ActivityThread: handleBindApplication()++ app=app.revanced.manager.flutter
01-13 16:46:38.734 31709 31709 D CompatibilityChangeReporter: Compat change id reported: 171979766; UID 10420; state: ENABLED
01-13 16:46:38.759 31709 31709 V GraphicsEnvironment: ANGLE Developer option for 'app.revanced.manager.flutter' set to: 'default'
01-13 16:46:38.759 31709 31709 V GraphicsEnvironment: App is not on the allowlist for updatable production driver.
01-13 16:46:38.763 31709 31709 D LoadedApk: LoadedApk::makeApplication() appContext.mOpPackageName=app.revanced.manager.flutter appContext.mBasePackageName=app.revanced.manager.flutter
01-13 16:46:38.763 31709 31709 D NetworkSecurityConfig: No Network Security Config specified, using platform default
01-13 16:46:38.767 31709 31709 D NetworkSecurityConfig: No Network Security Config specified, using platform default
01-13 16:46:38.781 31709 31709 D ActivityThread: handleBindApplication() --
01-13 16:46:38.791 31709 31772 D OpenGLRenderer: RenderThread::requireGlContext()
01-13 16:46:38.803 31709 31772 D OpenGLRenderer: RenderThread::setGrContext()
01-13 16:46:38.929 31709 31709 D FlutterFileDialogPlugin: onAttachedToEngine - IN
01-13 16:46:38.929 31709 31709 D FlutterFileDialogPlugin: doOnAttachedToEngine - IN
01-13 16:46:38.929 31709 31709 D FlutterFileDialogPlugin: doOnAttachedToEngine - OUT
01-13 16:46:38.929 31709 31709 D FlutterFileDialogPlugin: onAttachedToEngine - OUT
01-13 16:46:38.929 31709 31709 D FlutterFileDialogPlugin: onAttachedToActivity
01-13 16:46:38.929 31709 31709 D FlutterFileDialogPlugin: doOnAttachedToActivity - IN
01-13 16:46:38.929 31709 31709 D FlutterFileDialogPlugin: doOnAttachedToActivity - OUT
01-13 16:46:38.957 31709 31709 I DecorView: [INFO] isPopOver=false, config=true
01-13 16:46:38.957 31709 31709 I DecorView: updateCaptionType >> DecorView@28ce8da[], isFloating=false, isApplication=true, hasWindowControllerCallback=true, hasWindowDecorCaption=false
01-13 16:46:38.957 31709 31709 D DecorView: setCaptionType = 0, this = DecorView@28ce8da[]
01-13 16:46:38.958 31709 31709 I DecorView: getCurrentDensityDpi: from real metrics. densityDpi=420 msg=resources_loaded
01-13 16:46:38.967 31709 31709 I DecorView: notifyKeepScreenOnChanged: keepScreenOn=false
01-13 16:46:38.971 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: setView = com.android.internal.policy.DecorView@28ce8da TM=true
01-13 16:46:38.985 31709 31709 I SurfaceView@3c89293: onWindowVisibilityChanged(0) true io.flutter.embedding.android.FlutterSurfaceView{3c89293 V.E...... ......I. 0,0-0,0} of ViewRootImpl@a60b8a6[MainActivity]
01-13 16:46:38.994 31709 31709 D InsetsSourceConsumer: ensureControlAlpha: for ITYPE_NAVIGATION_BAR on app.revanced.manager.flutter/app.revanced.manager.flutter.MainActivity
01-13 16:46:38.995 31709 31709 D InsetsSourceConsumer: ensureControlAlpha: for ITYPE_STATUS_BAR on app.revanced.manager.flutter/app.revanced.manager.flutter.MainActivity
01-13 16:46:38.995 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: Relayout returned: old=(0,0,1080,2280) new=(0,0,1080,2280) req=(1080,2280)0 dur=8 res=0x7 s={true 489332925344} ch=true fn=-1
01-13 16:46:38.996 31709 31709 I SurfaceView@3c89293: windowStopped(false) true io.flutter.embedding.android.FlutterSurfaceView{3c89293 V.E...... ......ID 0,0-1080,2154} of ViewRootImpl@a60b8a6[MainActivity]
01-13 16:46:38.997 31709 31772 D hw-ProcessState: Binder ioctl to enable oneway spam detection failed: Invalid argument
01-13 16:46:38.998 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: [DP] dp(1) 1 android.view.ViewRootImpl.reportNextDraw:11442 android.view.ViewRootImpl.performTraversals:4198 android.view.ViewRootImpl.doTraversal:2924 
01-13 16:46:38.998 31709 31772 D OpenGLRenderer: eglCreateWindowSurface
01-13 16:46:39.002 31709 31709 I SurfaceView@3c89293: pST: sr = Rect(0, 0 - 1080, 2154) sw = 1080 sh = 2154
01-13 16:46:39.002 31709 31709 I SurfaceView@3c89293: onSSPAndSRT: pl = 0 pt = 0 sx = 1.0 sy = 1.0
01-13 16:46:39.002 31709 31709 I SurfaceView@3c89293: pST: mTmpTransaction.apply, mTmpTransaction = android.view.SurfaceControl$Transaction@c485851
01-13 16:46:39.002 31709 31709 I SurfaceView@3c89293: updateSurface: mVisible = true mSurface.isValid() = true
01-13 16:46:39.002 31709 31709 I SurfaceView@3c89293: updateSurface: mSurfaceCreated = false surfaceChanged = true visibleChanged = true
01-13 16:46:39.002 31709 31709 I SurfaceView@3c89293: surfaceCreated 1 #8 io.flutter.embedding.android.FlutterSurfaceView{3c89293 V.E...... ......ID 0,0-1080,2154}
01-13 16:46:39.005 31709 31709 I SurfaceView@3c89293: surfaceChanged (1080,2154) 1 #8 io.flutter.embedding.android.FlutterSurfaceView{3c89293 V.E...... ......ID 0,0-1080,2154}
01-13 16:46:39.005 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: [DP] dp(2) 1 android.view.SurfaceView.updateSurface:1375 android.view.SurfaceView.lambda$new$1$SurfaceView:254 android.view.SurfaceView$$ExternalSyntheticLambda2.onPreDraw:2 
01-13 16:46:39.005 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: [DP] pdf(1) 1 android.view.SurfaceView.notifyDrawFinished:599 android.view.SurfaceView.performDrawFinished:586 android.view.SurfaceView.$r8$lambda$st27mCkd9jfJkTrN_P3qIGKX6NY:0 
01-13 16:46:39.005 31709 31709 D ViewRootImpl@a60b8a6[MainActivity]: pendingDrawFinished. Waiting on draw reported mDrawsNeededToReport=1
01-13 16:46:39.851 31709 31787 I flutter : dynamic_color: Core palette detected.
01-13 16:46:39.896 31709 31788 I Gralloc4: mapper 4.x is not supported
01-13 16:46:39.897 31709 31788 W Gralloc3: mapper 3.x is not supported
01-13 16:46:39.898 31709 31788 I gralloc : Arm Module v1.0
01-13 16:46:39.900 31709 31788 W Gralloc4: allocator 4.x is not supported
01-13 16:46:39.900 31709 31788 W Gralloc3: allocator 3.x is not supported
01-13 16:46:39.966 31709 31709 D ViewRootImpl@a60b8a6[MainActivity]: Creating frameDrawingCallback nextDrawUseBlastSync=false reportNextDraw=true hasBlurUpdates=false
01-13 16:46:39.967 31709 31709 D ViewRootImpl@a60b8a6[MainActivity]: Creating frameCompleteCallback
01-13 16:46:39.970 31709 32376 I SurfaceView@3c89293: uSP: rtp = Rect(0, 0 - 1080, 2154) rtsw = 1080 rtsh = 2154
01-13 16:46:39.970 31709 32376 I SurfaceView@3c89293: onSSPAndSRT: pl = 0 pt = 0 sx = 1.0 sy = 1.0
01-13 16:46:39.970 31709 32376 I SurfaceView@3c89293: aOrMT: uB = true t = android.view.SurfaceControl$Transaction@144874 fN = 1 android.view.SurfaceView.access$500:124 android.view.SurfaceView$SurfaceViewPositionUpdateListener.positionChanged:1728 android.graphics.RenderNode$CompositePositionUpdateListener.positionChanged:319 
01-13 16:46:39.970 31709 32376 I SurfaceView@3c89293: aOrMT: vR.mWNT, vR = ViewRootImpl@a60b8a6[MainActivity]
01-13 16:46:39.970 31709 32376 I ViewRootImpl@a60b8a6[MainActivity]: mWNT: t = android.view.SurfaceControl$Transaction@144874 fN = 1 android.view.SurfaceView.applyOrMergeTransaction:1628 android.view.SurfaceView.access$500:124 android.view.SurfaceView$SurfaceViewPositionUpdateListener.positionChanged:1728 
01-13 16:46:39.970 31709 32376 I ViewRootImpl@a60b8a6[MainActivity]: mWNT: merge t to BBQ
01-13 16:46:39.970 31709 32376 D ViewRootImpl@a60b8a6[MainActivity]: Received frameDrawingCallback frameNum=1. Creating transactionCompleteCallback=false
01-13 16:46:39.987 31709 31772 D ViewRootImpl@a60b8a6[MainActivity]: Received frameCompleteCallback  lastAcquiredFrameNum=1 lastAttemptedDrawFrameNum=1
01-13 16:46:39.989 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: [DP] pdf(0) 1 android.view.ViewRootImpl.lambda$addFrameCompleteCallbackIfNeeded$3$ViewRootImpl:5000 android.view.ViewRootImpl$$ExternalSyntheticLambda16.run:6 android.os.Handler.handleCallback:938 
01-13 16:46:39.990 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: [DP] rdf()
01-13 16:46:39.990 31709 31709 D ViewRootImpl@a60b8a6[MainActivity]: reportDrawFinished (fn: -1) 
01-13 16:46:40.002 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: MSG_WINDOW_FOCUS_CHANGED 1 1
01-13 16:46:40.003 31709 31709 D InputMethodManager: startInputInner - Id : 0
01-13 16:46:40.003 31709 31709 I InputMethodManager: startInputInner - mService.startInputOrWindowGainedFocus
01-13 16:46:40.126 31709 31709 D InsetsSourceConsumer: ensureControlAlpha: for ITYPE_NAVIGATION_BAR on app.revanced.manager.flutter/app.revanced.manager.flutter.MainActivity
01-13 16:46:40.127 31709 31709 D InsetsSourceConsumer: ensureControlAlpha: for ITYPE_STATUS_BAR on app.revanced.manager.flutter/app.revanced.manager.flutter.MainActivity
01-13 16:46:40.185 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: MSG_WINDOW_FOCUS_CHANGED 0 1
01-13 16:46:41.351 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: stopped(false) old=false
01-13 16:46:41.361 31709 31709 I DecorView: notifyKeepScreenOnChanged: keepScreenOn=false
01-13 16:46:41.361 31709 31709 D InsetsSourceConsumer: ensureControlAlpha: for ITYPE_NAVIGATION_BAR on app.revanced.manager.flutter/app.revanced.manager.flutter.MainActivity
01-13 16:46:41.362 31709 31709 D InsetsSourceConsumer: ensureControlAlpha: for ITYPE_STATUS_BAR on app.revanced.manager.flutter/app.revanced.manager.flutter.MainActivity
01-13 16:46:41.875 31709 31709 I Choreographer: Skipped 30 frames!  The application may be doing too much work on its main thread.
01-13 16:46:41.878 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: Relayout returned: old=(0,0,1080,2280) new=(0,0,1080,2280) req=(1080,2280)0 dur=3 res=0x1 s={true 489332925344} ch=false fn=2
01-13 16:46:41.878 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: updateBoundsLayer: t = android.view.SurfaceControl$Transaction@f6c102b sc = Surface(name=Bounds for - app.revanced.manager.flutter/app.revanced.manager.flutter.MainActivity@0)/@0xa322c88 frame = 2
01-13 16:46:41.878 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: mWNT: t = android.view.SurfaceControl$Transaction@f6c102b fN = 2 android.view.ViewRootImpl.prepareSurfaces:2783 android.view.ViewRootImpl.performTraversals:4029 android.view.ViewRootImpl.doTraversal:2924 
01-13 16:46:41.878 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: mWNT: merge t to BBQ
01-13 16:46:41.879 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: MSG_WINDOW_FOCUS_CHANGED 1 1
01-13 16:46:41.880 31709 31709 D InputMethodManager: startInputInner - Id : 0
01-13 16:46:41.880 31709 31709 I InputMethodManager: startInputInner - mService.startInputOrWindowGainedFocus
01-13 16:46:41.896 31709 31709 D InputMethodManager: startInputInner - Id : 0
01-13 16:46:41.953 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: MSG_WINDOW_FOCUS_CHANGED 0 1
01-13 16:46:43.316 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: stopped(false) old=false
01-13 16:46:43.322 31709 31709 I DecorView: notifyKeepScreenOnChanged: keepScreenOn=false
01-13 16:46:43.322 31709 31709 D InsetsSourceConsumer: ensureControlAlpha: for ITYPE_NAVIGATION_BAR on app.revanced.manager.flutter/app.revanced.manager.flutter.MainActivity
01-13 16:46:43.323 31709 31709 D InsetsSourceConsumer: ensureControlAlpha: for ITYPE_STATUS_BAR on app.revanced.manager.flutter/app.revanced.manager.flutter.MainActivity
01-13 16:46:43.335 31709 31709 I DecorView: notifyKeepScreenOnChanged: keepScreenOn=false
01-13 16:46:43.340 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: Relayout returned: old=(0,0,1080,2280) new=(0,0,1080,2280) req=(1080,2280)0 dur=3 res=0x1 s={true 489332925344} ch=false fn=3
01-13 16:46:43.341 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: updateBoundsLayer: t = android.view.SurfaceControl$Transaction@f6c102b sc = Surface(name=Bounds for - app.revanced.manager.flutter/app.revanced.manager.flutter.MainActivity@0)/@0xa322c88 frame = 3
01-13 16:46:43.341 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: mWNT: t = android.view.SurfaceControl$Transaction@f6c102b fN = 3 android.view.ViewRootImpl.prepareSurfaces:2783 android.view.ViewRootImpl.performTraversals:4029 android.view.ViewRootImpl.doTraversal:2924 
01-13 16:46:43.341 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: mWNT: merge t to BBQ
01-13 16:46:43.341 31709 31709 I SurfaceView@3c89293: pST: sr = Rect(0, 0 - 1080, 2280) sw = 1080 sh = 2280
01-13 16:46:43.341 31709 31709 I SurfaceView@3c89293: onSSPAndSRT: pl = 0 pt = 0 sx = 1.0 sy = 1.0
01-13 16:46:43.341 31709 31709 I SurfaceView@3c89293: pST: mTmpTransaction.apply, mTmpTransaction = android.view.SurfaceControl$Transaction@c485851
01-13 16:46:43.342 31709 31709 I SurfaceView@3c89293: updateSurface: mVisible = true mSurface.isValid() = true
01-13 16:46:43.342 31709 31709 I SurfaceView@3c89293: updateSurface: mSurfaceCreated = true surfaceChanged = false visibleChanged = false
01-13 16:46:43.342 31709 31709 I SurfaceView@3c89293: surfaceChanged (1080,2280) 1 #5 io.flutter.embedding.android.FlutterSurfaceView{3c89293 V.E...... ......ID 0,0-1080,2280}
01-13 16:46:43.344 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: [DP] dp(1) 1 android.view.SurfaceView.updateSurface:1375 android.view.SurfaceView.setFrame:675 android.view.View.layout:24458 
01-13 16:46:43.344 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: [DP] pdf(0) 1 android.view.SurfaceView.notifyDrawFinished:599 android.view.SurfaceView.performDrawFinished:586 android.view.SurfaceView.$r8$lambda$st27mCkd9jfJkTrN_P3qIGKX6NY:0 
01-13 16:46:43.344 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: [DP] rdf()
01-13 16:46:43.345 31709 31709 D ViewRootImpl@a60b8a6[MainActivity]: reportDrawFinished (fn: 3) 
01-13 16:46:43.345 31709 32375 I SurfaceView@3c89293: uSP: rtp = Rect(0, 0 - 1080, 2280) rtsw = 1080 rtsh = 2280
01-13 16:46:43.345 31709 32375 I SurfaceView@3c89293: onSSPAndSRT: pl = 0 pt = 0 sx = 1.0 sy = 1.0
01-13 16:46:43.345 31709 32375 I SurfaceView@3c89293: aOrMT: uB = true t = android.view.SurfaceControl$Transaction@f4ecff fN = 3 android.view.SurfaceView.access$500:124 android.view.SurfaceView$SurfaceViewPositionUpdateListener.positionChanged:1728 android.graphics.RenderNode$CompositePositionUpdateListener.positionChanged:319 
01-13 16:46:43.345 31709 32375 I SurfaceView@3c89293: aOrMT: vR.mWNT, vR = ViewRootImpl@a60b8a6[MainActivity]
01-13 16:46:43.345 31709 32375 I ViewRootImpl@a60b8a6[MainActivity]: mWNT: t = android.view.SurfaceControl$Transaction@f4ecff fN = 3 android.view.SurfaceView.applyOrMergeTransaction:1628 android.view.SurfaceView.access$500:124 android.view.SurfaceView$SurfaceViewPositionUpdateListener.positionChanged:1728 
01-13 16:46:43.345 31709 32375 I ViewRootImpl@a60b8a6[MainActivity]: mWNT: merge t to BBQ
01-13 16:46:43.352 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: MSG_WINDOW_FOCUS_CHANGED 1 1
01-13 16:46:43.353 31709 31709 D InputMethodManager: startInputInner - Id : 0
01-13 16:46:43.353 31709 31709 I InputMethodManager: startInputInner - mService.startInputOrWindowGainedFocus
01-13 16:46:43.931 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: ViewPostIme pointer 0
01-13 16:46:44.013 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: ViewPostIme pointer 1
01-13 16:46:45.699 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: ViewPostIme pointer 0
01-13 16:46:45.699 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: ViewPostIme pointer 1
01-13 16:46:46.436 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: ViewPostIme pointer 0
01-13 16:46:46.491 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: ViewPostIme pointer 1
01-13 16:46:46.536 31709 31709 I AppIconSolution: night mode is changed to true
01-13 16:46:46.640 31709 31731 D InputTransport: Input channel destroyed: 'ClientS', fd=123
01-13 16:46:48.498 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: ViewPostIme pointer 0
01-13 16:46:48.559 31709 31709 I ViewRootImpl@a60b8a6[MainActivity]: ViewPostIme pointer 1
01-13 16:46:48.562 31709 31709 D FlutterFileDialogPlugin: onMethodCall - IN , method=pickFile
01-13 16:46:48.562 31709 31709 D FlutterFileDialogPlugin: createFileDialog - IN
01-13 16:46:48.562 31709 31709 D FlutterFileDialogPlugin: createFileDialog - OUT
01-13 16:46:48.562 31709 31709 D FileDialog: pickFile - IN, fileExtensionsFilter=[Ljava.lang.String;@22a0653, mimeTypesFilter=null, localOnly=false, copyFileToCacheDir=true
01-13 16:46:48.565 31709 31709 E MethodChannel#flutter_file_dialog: Failed to handle method call
01-13 16:46:48.565 31709 31709 E MethodChannel#flutter_file_dialog: android.content.ActivityNotFoundException: No Activity found to handle Intent { act=android.intent.action.OPEN_DOCUMENT cat=[android.intent.category.OPENABLE] typ=*/* }
01-13 16:46:48.565 31709 31709 E MethodChannel#flutter_file_dialog: 	at android.app.Instrumentation.checkStartActivityResult(Instrumentation.java:2100)
01-13 16:46:48.565 31709 31709 E MethodChannel#flutter_file_dialog: 	at android.app.Instrumentation.execStartActivity(Instrumentation.java:1747)
01-13 16:46:48.565 31709 31709 E MethodChannel#flutter_file_dialog: 	at android.app.Activity.startActivityForResult(Activity.java:5473)
01-13 16:46:48.565 31709 31709 E MethodChannel#flutter_file_dialog: 	at android.app.Activity.startActivityForResult(Activity.java:5431)
01-13 16:46:48.565 31709 31709 E MethodChannel#flutter_file_dialog: 	at com.kineapps.flutter_file_dialog.FileDialog.pickFile(FileDialog.kt:94)
01-13 16:46:48.565 31709 31709 E MethodChannel#flutter_file_dialog: 	at com.kineapps.flutter_file_dialog.FlutterFileDialogPlugin.onMethodCall(FlutterFileDialogPlugin.kt:135)
01-13 16:46:48.565 31709 31709 E MethodChannel#flutter_file_dialog: 	at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:267)
01-13 16:46:48.565 31709 31709 E MethodChannel#flutter_file_dialog: 	at io.flutter.embedding.engine.dart.DartMessenger.invokeHandler(DartMessenger.java:295)
01-13 16:46:48.565 31709 31709 E MethodChannel#flutter_file_dialog: 	at io.flutter.embedding.engine.dart.DartMessenger.lambda$dispatchMessageToQueue$0$io-flutter-embedding-engine-dart-DartMessenger(DartMessenger.java:322)
01-13 16:46:48.565 31709 31709 E MethodChannel#flutter_file_dialog: 	at io.flutter.embedding.engine.dart.DartMessenger$$ExternalSyntheticLambda0.run(Unknown Source:12)
01-13 16:46:48.565 31709 31709 E MethodChannel#flutter_file_dialog: 	at android.os.Handler.handleCallback(Handler.java:938)
01-13 16:46:48.565 31709 31709 E MethodChannel#flutter_file_dialog: 	at android.os.Handler.dispatchMessage(Handler.java:99)
01-13 16:46:48.565 31709 31709 E MethodChannel#flutter_file_dialog: 	at android.os.Looper.loopOnce(Looper.java:226)
01-13 16:46:48.565 31709 31709 E MethodChannel#flutter_file_dialog: 	at android.os.Looper.loop(Looper.java:313)
01-13 16:46:48.565 31709 31709 E MethodChannel#flutter_file_dialog: 	at android.app.ActivityThread.main(ActivityThread.java:8751)
01-13 16:46:48.565 31709 31709 E MethodChannel#flutter_file_dialog: 	at java.lang.reflect.Method.invoke(Native Method)
01-13 16:46:48.565 31709 31709 E MethodChannel#flutter_file_dialog: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
01-13 16:46:48.565 31709 31709 E MethodChannel#flutter_file_dialog: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
01-13 16:46:48.566 31709 31709 D FileDialog: Cancelled
01-13 16:46:48.566 31709 31709 D AndroidRuntime: Shutting down VM
--------- beginning of crash
01-13 16:46:48.567 31709 31709 E AndroidRuntime: FATAL EXCEPTION: main
01-13 16:46:48.567 31709 31709 E AndroidRuntime: Process: app.revanced.manager.flutter, PID: 31709
01-13 16:46:48.567 31709 31709 E AndroidRuntime: java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=19111, result=0, data=null} to activity {app.revanced.manager.flutter/app.revanced.manager.flutter.MainActivity}: java.lang.IllegalStateException: Reply already submitted
01-13 16:46:48.567 31709 31709 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5946)
01-13 16:46:48.567 31709 31709 E AndroidRuntime: 	at android.app.ActivityThread.handleSendResult(ActivityThread.java:5985)
01-13 16:46:48.567 31709 31709 E AndroidRuntime: 	at android.app.servertransaction.ActivityResultItem.execute(ActivityResultItem.java:54)
01-13 16:46:48.567 31709 31709 E AndroidRuntime: 	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
01-13 16:46:48.567 31709 31709 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
01-13 16:46:48.567 31709 31709 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
01-13 16:46:48.567 31709 31709 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2443)
01-13 16:46:48.567 31709 31709 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
01-13 16:46:48.567 31709 31709 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:226)
01-13 16:46:48.567 31709 31709 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:313)
01-13 16:46:48.567 31709 31709 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8751)
01-13 16:46:48.567 31709 31709 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
01-13 16:46:48.567 31709 31709 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
01-13 16:46:48.567 31709 31709 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
01-13 16:46:48.567 31709 31709 E AndroidRuntime: Caused by: java.lang.IllegalStateException: Reply already submitted
01-13 16:46:48.567 31709 31709 E AndroidRuntime: 	at io.flutter.embedding.engine.dart.DartMessenger$Reply.reply(DartMessenger.java:435)
01-13 16:46:48.567 31709 31709 E AndroidRuntime: 	at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler$1.success(MethodChannel.java:272)
01-13 16:46:48.567 31709 31709 E AndroidRuntime: 	at com.kineapps.flutter_file_dialog.FileDialog.finishSuccessfully(FileDialog.kt:364)
01-13 16:46:48.567 31709 31709 E AndroidRuntime: 	at com.kineapps.flutter_file_dialog.FileDialog.onActivityResult(FileDialog.kt:194)
01-13 16:46:48.567 31709 31709 E AndroidRuntime: 	at io.flutter.embedding.engine.FlutterEngineConnectionRegistry$FlutterEngineActivityPluginBinding.onActivityResult(FlutterEngineConnectionRegistry.java:813)
01-13 16:46:48.567 31709 31709 E AndroidRuntime: 	at io.flutter.embedding.engine.FlutterEngineConnectionRegistry.onActivityResult(FlutterEngineConnectionRegistry.java:432)
01-13 16:46:48.567 31709 31709 E AndroidRuntime: 	at io.flutter.embedding.android.FlutterActivityAndFragmentDelegate.onActivityResult(FlutterActivityAndFragmentDelegate.java:853)
01-13 16:46:48.567 31709 31709 E AndroidRuntime: 	at io.flutter.embedding.android.FlutterActivity.onActivityResult(FlutterActivity.java:884)
01-13 16:46:48.567 31709 31709 E AndroidRuntime: 	at android.app.Activity.dispatchActivityResult(Activity.java:8659)
01-13 16:46:48.567 31709 31709 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5939)
01-13 16:46:48.567 31709 31709 E AndroidRuntime: 	... 13 more
01-13 16:46:48.616 31709 31709 I Process : Sending signal. PID: 31709 SIG: 9
01-13 16:46:50.464   906   906 E manager.flutte: Not starting debugger since process cannot load the jdwp agent.
01-13 16:46:50.466   906   906 E USNET   : USNET: appName: app.revanced.manager.flutter
01-13 16:46:50.467   906   906 D ProcessState: Binder ioctl to enable oneway spam detection failed: Invalid argument
01-13 16:46:50.471   906   906 D ActivityThread: setConscryptValidator
01-13 16:46:50.472   906   906 D ActivityThread: setConscryptValidator - put
01-13 16:46:50.492   906   906 D ActivityThread: handleBindApplication()++ app=app.revanced.manager.flutter
01-13 16:46:50.492   906   906 D CompatibilityChangeReporter: Compat change id reported: 171979766; UID 10420; state: ENABLED
01-13 16:46:50.510   906   906 V GraphicsEnvironment: ANGLE Developer option for 'app.revanced.manager.flutter' set to: 'default'
01-13 16:46:50.510   906   906 V GraphicsEnvironment: App is not on the allowlist for updatable production driver.
01-13 16:46:50.512   906   906 D LoadedApk: LoadedApk::makeApplication() appContext.mOpPackageName=app.revanced.manager.flutter appContext.mBasePackageName=app.revanced.manager.flutter
01-13 16:46:50.512   906   906 D NetworkSecurityConfig: No Network Security Config specified, using platform default
01-13 16:46:50.514   906   906 D NetworkSecurityConfig: No Network Security Config specified, using platform default
01-13 16:46:50.519   906   906 D ActivityThread: handleBindApplication() --
01-13 16:46:50.523   906   934 D OpenGLRenderer: RenderThread::requireGlContext()
01-13 16:46:50.532   906   934 D OpenGLRenderer: RenderThread::setGrContext()
01-13 16:46:50.579   906   906 D FlutterFileDialogPlugin: onAttachedToEngine - IN
01-13 16:46:50.579   906   906 D FlutterFileDialogPlugin: doOnAttachedToEngine - IN
01-13 16:46:50.579   906   906 D FlutterFileDialogPlugin: doOnAttachedToEngine - OUT
01-13 16:46:50.579   906   906 D FlutterFileDialogPlugin: onAttachedToEngine - OUT
01-13 16:46:50.579   906   906 D FlutterFileDialogPlugin: onAttachedToActivity
01-13 16:46:50.579   906   906 D FlutterFileDialogPlugin: doOnAttachedToActivity - IN
01-13 16:46:50.579   906   906 D FlutterFileDialogPlugin: doOnAttachedToActivity - OUT
01-13 16:46:50.599   906   906 I DecorView: [INFO] isPopOver=false, config=true
01-13 16:46:50.599   906   906 I DecorView: updateCaptionType >> DecorView@78dc10b[], isFloating=false, isApplication=true, hasWindowControllerCallback=true, hasWindowDecorCaption=false
01-13 16:46:50.599   906   906 D DecorView: setCaptionType = 0, this = DecorView@78dc10b[]
01-13 16:46:50.599   906   906 I DecorView: getCurrentDensityDpi: from real metrics. densityDpi=420 msg=resources_loaded
01-13 16:46:50.607   906   906 I DecorView: notifyKeepScreenOnChanged: keepScreenOn=false
01-13 16:46:50.609   906   906 I ViewRootImpl@9d562e7[MainActivity]: setView = com.android.internal.policy.DecorView@78dc10b TM=true
01-13 16:46:50.625   906   906 I SurfaceView@9c162d0: onWindowVisibilityChanged(0) true io.flutter.embedding.android.FlutterSurfaceView{9c162d0 V.E...... ......I. 0,0-0,0} of ViewRootImpl@9d562e7[MainActivity]
01-13 16:46:50.629   906   906 I ViewRootImpl@9d562e7[MainActivity]: Relayout returned: old=(0,0,1080,2280) new=(0,0,1080,2280) req=(1080,2280)0 dur=4 res=0x7 s={true 489332925344} ch=true fn=-1
01-13 16:46:50.630   906   906 I SurfaceView@9c162d0: windowStopped(false) true io.flutter.embedding.android.FlutterSurfaceView{9c162d0 V.E...... ......ID 0,0-1080,2154} of ViewRootImpl@9d562e7[MainActivity]
01-13 16:46:50.630   906   906 I ViewRootImpl@9d562e7[MainActivity]: [DP] dp(1) 1 android.view.ViewRootImpl.reportNextDraw:11442 android.view.ViewRootImpl.performTraversals:4198 android.view.ViewRootImpl.doTraversal:2924 
01-13 16:46:50.631   906   934 D hw-ProcessState: Binder ioctl to enable oneway spam detection failed: Invalid argument
01-13 16:46:50.632   906   934 D OpenGLRenderer: eglCreateWindowSurface
01-13 16:46:50.632   906   906 I SurfaceView@9c162d0: pST: sr = Rect(0, 0 - 1080, 2154) sw = 1080 sh = 2154
01-13 16:46:50.632   906   906 I SurfaceView@9c162d0: onSSPAndSRT: pl = 0 pt = 0 sx = 1.0 sy = 1.0
01-13 16:46:50.633   906   906 I SurfaceView@9c162d0: pST: mTmpTransaction.apply, mTmpTransaction = android.view.SurfaceControl$Transaction@7eb09b7
01-13 16:46:50.633   906   906 I SurfaceView@9c162d0: updateSurface: mVisible = true mSurface.isValid() = true
01-13 16:46:50.633   906   906 I SurfaceView@9c162d0: updateSurface: mSurfaceCreated = false surfaceChanged = true visibleChanged = true
01-13 16:46:50.633   906   906 I SurfaceView@9c162d0: surfaceCreated 1 #8 io.flutter.embedding.android.FlutterSurfaceView{9c162d0 V.E...... ......ID 0,0-1080,2154}
01-13 16:46:50.635   906   906 I SurfaceView@9c162d0: surfaceChanged (1080,2154) 1 #8 io.flutter.embedding.android.FlutterSurfaceView{9c162d0 V.E...... ......ID 0,0-1080,2154}
01-13 16:46:50.635   906   906 I ViewRootImpl@9d562e7[MainActivity]: [DP] dp(2) 1 android.view.SurfaceView.updateSurface:1375 android.view.SurfaceView.lambda$new$1$SurfaceView:254 android.view.SurfaceView$$ExternalSyntheticLambda2.onPreDraw:2 
01-13 16:46:50.635   906   906 I ViewRootImpl@9d562e7[MainActivity]: [DP] pdf(1) 1 android.view.SurfaceView.notifyDrawFinished:599 android.view.SurfaceView.performDrawFinished:586 android.view.SurfaceView.$r8$lambda$st27mCkd9jfJkTrN_P3qIGKX6NY:0 
01-13 16:46:50.635   906   906 D ViewRootImpl@9d562e7[MainActivity]: pendingDrawFinished. Waiting on draw reported mDrawsNeededToReport=1
01-13 16:46:51.048   906   906 D InsetsSourceConsumer: ensureControlAlpha: for ITYPE_NAVIGATION_BAR on app.revanced.manager.flutter/app.revanced.manager.flutter.MainActivity
01-13 16:46:51.049   906   906 D InsetsSourceConsumer: ensureControlAlpha: for ITYPE_STATUS_BAR on app.revanced.manager.flutter/app.revanced.manager.flutter.MainActivity
01-13 16:46:51.468   906   948 I flutter : dynamic_color: Core palette detected.
01-13 16:46:51.489   906   906 I DecorView: notifyKeepScreenOnChanged: keepScreenOn=false
01-13 16:46:51.493   906   906 I ViewRootImpl@9d562e7[MainActivity]: Relayout returned: old=(0,0,1080,2280) new=(0,0,1080,2280) req=(1080,2280)0 dur=3 res=0x3 s={true 489332925344} ch=false fn=1
01-13 16:46:51.494   906   906 I SurfaceView@9c162d0: pST: sr = Rect(0, 0 - 1080, 2280) sw = 1080 sh = 2280
01-13 16:46:51.494   906   906 I SurfaceView@9c162d0: onSSPAndSRT: pl = 0 pt = 0 sx = 1.0 sy = 1.0
01-13 16:46:51.494   906   906 I SurfaceView@9c162d0: pST: mTmpTransaction.apply, mTmpTransaction = android.view.SurfaceControl$Transaction@7eb09b7
01-13 16:46:51.495   906   906 I SurfaceView@9c162d0: updateSurface: mVisible = true mSurface.isValid() = true
01-13 16:46:51.495   906   906 I SurfaceView@9c162d0: updateSurface: mSurfaceCreated = true surfaceChanged = false visibleChanged = false
01-13 16:46:51.495   906   906 I SurfaceView@9c162d0: surfaceChanged (1080,2280) 1 #5 io.flutter.embedding.android.FlutterSurfaceView{9c162d0 V.E...... ......ID 0,0-1080,2280}
01-13 16:46:51.495   906   906 I ViewRootImpl@9d562e7[MainActivity]: [DP] dp(2) 1 android.view.SurfaceView.updateSurface:1375 android.view.SurfaceView.setFrame:675 android.view.View.layout:24458 
01-13 16:46:51.495   906   906 I ViewRootImpl@9d562e7[MainActivity]: [DP] pdf(1) 1 android.view.SurfaceView.notifyDrawFinished:599 android.view.SurfaceView.performDrawFinished:586 android.view.SurfaceView.$r8$lambda$st27mCkd9jfJkTrN_P3qIGKX6NY:0 
01-13 16:46:51.495   906   906 D ViewRootImpl@9d562e7[MainActivity]: pendingDrawFinished. Waiting on draw reported mDrawsNeededToReport=1
01-13 16:46:51.522   906   949 I Gralloc4: mapper 4.x is not supported
01-13 16:46:51.523   906   949 W Gralloc3: mapper 3.x is not supported
01-13 16:46:51.524   906   949 I gralloc : Arm Module v1.0
01-13 16:46:51.525   906   949 W Gralloc4: allocator 4.x is not supported
01-13 16:46:51.525   906   949 W Gralloc3: allocator 3.x is not supported
01-13 16:46:51.539   906   906 D ViewRootImpl@9d562e7[MainActivity]: Creating frameDrawingCallback nextDrawUseBlastSync=false reportNextDraw=true hasBlurUpdates=false
01-13 16:46:51.539   906   906 D ViewRootImpl@9d562e7[MainActivity]: Creating frameCompleteCallback
01-13 16:46:51.540   906  2121 I SurfaceView@9c162d0: uSP: rtp = Rect(0, 0 - 1080, 2280) rtsw = 1080 rtsh = 2280
01-13 16:46:51.540   906  2121 I SurfaceView@9c162d0: onSSPAndSRT: pl = 0 pt = 0 sx = 1.0 sy = 1.0
01-13 16:46:51.540   906  2121 I SurfaceView@9c162d0: aOrMT: uB = true t = android.view.SurfaceControl$Transaction@144874 fN = 1 android.view.SurfaceView.access$500:124 android.view.SurfaceView$SurfaceViewPositionUpdateListener.positionChanged:1728 android.graphics.RenderNode$CompositePositionUpdateListener.positionChanged:319 
01-13 16:46:51.540   906  2121 I SurfaceView@9c162d0: aOrMT: vR.mWNT, vR = ViewRootImpl@9d562e7[MainActivity]
01-13 16:46:51.540   906  2121 I ViewRootImpl@9d562e7[MainActivity]: mWNT: t = android.view.SurfaceControl$Transaction@144874 fN = 1 android.view.SurfaceView.applyOrMergeTransaction:1628 android.view.SurfaceView.access$500:124 android.view.SurfaceView$SurfaceViewPositionUpdateListener.positionChanged:1728 
01-13 16:46:51.540   906  2120 D ViewRootImpl@9d562e7[MainActivity]: Received frameDrawingCallback frameNum=1. Creating transactionCompleteCallback=false
01-13 16:46:51.540   906  2121 I ViewRootImpl@9d562e7[MainActivity]: mWNT: merge t to BBQ
01-13 16:46:51.545   906   934 D ViewRootImpl@9d562e7[MainActivity]: Received frameCompleteCallback  lastAcquiredFrameNum=1 lastAttemptedDrawFrameNum=1
01-13 16:46:51.546   906   906 I ViewRootImpl@9d562e7[MainActivity]: [DP] pdf(0) 1 android.view.ViewRootImpl.lambda$addFrameCompleteCallbackIfNeeded$3$ViewRootImpl:5000 android.view.ViewRootImpl$$ExternalSyntheticLambda16.run:6 android.os.Handler.handleCallback:938 
01-13 16:46:51.546   906   906 I ViewRootImpl@9d562e7[MainActivity]: [DP] rdf()
01-13 16:46:51.546   906   906 D ViewRootImpl@9d562e7[MainActivity]: reportDrawFinished (fn: 1) 
01-13 16:46:51.558   906   906 I ViewRootImpl@9d562e7[MainActivity]: MSG_WINDOW_FOCUS_CHANGED 1 1
01-13 16:46:51.559   906   906 D InputMethodManager: startInputInner - Id : 0
01-13 16:46:51.559   906   906 I InputMethodManager: startInputInner - mService.startInputOrWindowGainedFocus
01-13 16:46:51.563   906   906 D InputMethodManager: startInputInner - Id : 0
01-13 16:46:52.451   906   906 I ViewRootImpl@9d562e7[MainActivity]: ViewPostIme pointer 0
01-13 16:46:52.453   906   906 I ViewRootImpl@9d562e7[MainActivity]: ViewPostIme pointer 1
01-13 16:46:54.558   906   906 I ViewRootImpl@9d562e7[MainActivity]: ViewPostIme pointer 0
01-13 16:46:54.637   906   906 I ViewRootImpl@9d562e7[MainActivity]: ViewPostIme pointer 1
01-13 16:46:56.099   906   906 I ViewRootImpl@9d562e7[MainActivity]: ViewPostIme pointer 0
01-13 16:46:56.172   906   906 I ViewRootImpl@9d562e7[MainActivity]: ViewPostIme pointer 1
```

### Acknowledgements

- [X] This request is not a duplicate of an existing issue.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] The issue is solely related to the ReVanced Manager

**Comment by s0nnenb0y** [comment]
I have the exact same issue, I can't patch YouTube anymore because it instantly crashes if I try to add the apk from my phone's storage.

**Comment by Spydron3** [comment]
I have the same problem with an Android 9 device.

**Comment by ILoveOpenSourceApplications** [comment]
Hopefully this can be closed as completed with the merge of #1768.

**Comment by GKid94** [comment]
Same
![Screenshot_2024-05-08-22-14-49-004_com miui home](https://github.com/ReVanced/revanced-manager/assets/18757054/b4c72731-feff-439e-9f4e-89556067ae9a)
[revanced-manager_logcat_20240508222024423525.log](https://github.com/ReVanced/revanced-manager/files/15254429/revanced-manager_logcat_20240508222024423525.log)

**Comment by oSumAtrIX** [comment]
https://github.com/ReVanced/revanced-manager/issues/1883
